src/fu-tool.c

```diff
@@ -1599,6 +1599,35 @@ fu_util_get_history (FuUtilPrivate *priv, gchar **values, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_util_get_remotes (FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(GNode) root = g_node_new (NULL);
+	g_autoptr(GPtrArray) remotes = NULL;
+	g_autofree gchar *title = fu_util_get_tree_title (priv);
+
+	/* load engine */
+	if (!fu_util_start_engine (priv, FU_ENGINE_LOAD_FLAG_NONE, error))
+		return FALSE;
+
+	remotes = fu_engine_get_remotes (priv->engine, error);
+	if (remotes == NULL)
+		return FALSE;
+	if (remotes->len == 0) {
+		/* TRANSLATORS: no repositories to download from */
+		g_print ("%s\n", _("No remotes available"));
+		return TRUE;
```

Comment:
Shouldn't this return nothing to do instead?

Reply:
Probably yes. Will fix.